src/components/SearchResults/index.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { HashLink as Link } from 'react-router-hash-link';
+import { Icon } from '@edx/paragon';
+import './SearchResults.scss';
+import {
+  RICH_TEXT,
+  RAW_HTML,
+  PDF,
+  XBLOCK_VIDEO,
+  IMAGE,
+} from '../../data/constants/contentTypes';
+
+import {
+  FILTER_ID_ALL,
+  FILTER_ID_IMAGES,
+  FILTER_ID_DOCUMENTS,
+  FILTER_ID_VIDEOS,
+} from '../../data/constants/filterTypes';
+
+const NUM_HIGHLIGHTS_DISPLAY = 3;
+
+const BreadCrumb = ({
+  breadcrumbs,
+}) => (
+  <div className="breadcrumb">
+    {
+    breadcrumbs.map(breadcrumb => (
+      <div key={breadcrumb} className="result-breadcrumb">
+        {breadcrumb}
+      </div>
+    ))
+  }
+  </div>
+);
+
+const AdditionalHighlights = ({
+  total,
+}) => (
+  total > NUM_HIGHLIGHTS_DISPLAY &&
+    <div className="additional-highlights">{`+ ${total - NUM_HIGHLIGHTS_DISPLAY} more instances`}</div>
+);
+
+const SearchHighlight = ({
+  highlights,
+}) => (
+  highlights.map(highlight => (
+    <div
+      key={highlight}
+      className="result-highlight"
+      dangerouslySetInnerHTML={{ __html: highlight }} // eslint-disable-line react/no-danger
+    />
+  ))
+);
+
+const SearchItem = ({
+  data,
+}) => (
+  <div className="search-result-card">
+    <Link className="card-link" to={`/${data.journal_about_page_id}/pages/${data.page_id}#${data.span_id}`} >
+
+      <div className="result-icon">
+        <Icon
+          className={
+            classNames({
+              fa: true,
+              'fa-book': data.block_type === RICH_TEXT || data.block_type === RAW_HTML,
+              'fa-file-video-o': data.block_type === XBLOCK_VIDEO,
+              'fa-picture-o': data.block_type === IMAGE,
+              'fa-file-pdf-o': data.block_type === PDF,
+            }).split(' ')
+          }
+        />
+        <span className="result-title">{data.block_title}</span>
+      </div>
+
+      <SearchHighlight highlights={data.highlights.slice(0, NUM_HIGHLIGHTS_DISPLAY)} />
+      <AdditionalHighlights total={data.highlights.length} />
+      <BreadCrumb breadcrumbs={data.breadcrumbs} />
+
+    </Link>
+  </div>
+);
+
+class SearchResults extends React.Component {
+  constructor(props) {
+    super(props);
+    this.formatKey = this.formatKey.bind(this);
```

Comment:
NIT: You don't need to bind this to methods that aren't being passed across components.